test/test-list-issues.ts

```
@@ -99,6 +103,20 @@ describe("list_issues", () => {
 
+  test("returns compact JSON text", async () => {
+    const result = await callListIssuesResult(
+      { project_id: TEST_PROJECT_ID },
+      {
+        GITLAB_API_URL: `${mockGitLabUrl}/api/v4`,
+        GITLAB_PERSONAL_ACCESS_TOKEN: MOCK_TOKEN,
+      }
+    );
+
+    assert.ok(result.text, "Tool response should include text content");
+    assert.doesNotMatch(result.text!, /\n\s+\"/, "Tool response JSON should not be pretty-printed");
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)